service/deploy.py

```diff
@@ -464,9 +465,27 @@ def wrap_script(script_text, script_name):
     * Chmod the file
     * Execute and redirect output to stdout/stderr to logfile.
     """
-    logfile = "/var/log/atmo/post_boot_scripts.log"
+    # logfile = "/var/log/atmo/post_boot_scripts.log"
     # kludge: weirdness without the str cast...
     script_text = str(script_text)
-    full_script_name = "./deploy_boot_script_%s.sh"
+    full_script_name = "./deploy_boot_script_%s.sh" % (slugify(script_name),)
     return ScriptDeployment(
         script_text, name=full_script_name)
+
+
+def _inject_env_script(username):
+    """
+    This is the 'raw script' that will be used to prepare the environment.
+    TODO: Find a better home for this. Probably use ansible for this.
+    """
+    return \
+        """#!/bin/bash -x
+atmo_user="%s"
+env_file="%s"
+env_vars=`cat $env_file`
+if [[ $env_vars == *"ATMO_USER="* ]]; then
+    sed -i "s/ATMO_USER=.*/ATMO_USER=\"$atmo_user\"/" $env_file
+else
+    echo "ATMO_USER=\"$atmo_user\"" >> $env_file
+fi
+""" % (username, "$HOME/.bashrc")
```

Comment:
I agree with the `TODO` here - we might want to make this a template, similarly to how we handle rendering into the [email feedback](https://github.com/iPlantCollaborativeOpenSource/atmosphere/tree/master/core/templates/core/email), etc. 

Maybe `core/templates/scripts`...